pyproject.toml

```diff
@@ -1,3 +1,64 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    { name = "SCCER-SoE" },
+    { name = "SED" },
+    { name = "ETHZ" },
+    { name = "Joseph Doetsch", email = "joseph.doetsch@erdw.ethz.ch" },
+    { name = "Linus Villiger", email = "linus.villiger@sed.ethz.ch" },
+    { name = "Thomas Haag" },
+    { name = "Sem Demir" },
+    { name = "DUGSeis Authors" },
```

Comment:
This list of authors reflects a very early version of the library. Should this be changed to or extended with the authors of the current paper?

Reply:
@mrosskopf can you adjust the package author list?